README.md

```diff
@@ -129,7 +129,7 @@ func rightDidClose()
 ```
 
 ## Requirements
-Requires Swift4.0 and iOS 9.0 and ARC.
```

Comment:
Maybe add in the Readme that this is a fork of Slide Menu Controller that was migrated to Swift 5 and is maintained by us to have clear documentation of why was this done.